doc/federation/federation.rst

```diff
@@ -0,0 +1,87 @@
+.. _federation/federation:
+
+========================
+The BinderHub Federation
+========================
+
+While it may seem like ``mybinder.org`` is a single website, it is
+in fact a **federation** of teams that deploy public BinderHubs to serve
+the community. This page lists the BinderHubs that currently help
+power ``mybinder.org``.
+
+Visiting ``mybinder.org`` will randomly redirect you to one
+of the following BinderHubs. You may also directly access a BinderHub
+by using its respective sub-domain in your URL.
+
+If your organization is interested in becoming part of the BinderHub
+federation, check out :ref:`federation/joining`.
+
+.. _federation/federation-list:
+
+Members of the BinderHub Federation
+===================================
+
+Here is a list of the current members of the BinderHub federation:
+
+.. include:: data-federation.txt
+
+
+.. _federation/joining:
+
+Joining the BinderHub Federation
+================================
+
+The Binder Project is an open community of developers as well as deployers
+of BinderHub. BinderHub is designed to be used by any organization for
+their own purposes. However, many members of the community also run
+the BinderHub deployments at ``mybinder.org`` as a large, free public service
+to the community.
```

Comment:
I'd remove this paragraph and instead put some of the info in the next one which I think should be the lead one as it answers the question "what is this federation you speak of?"